_posts/2023-05-23-GUIs-in-PyQt5.md

```diff
@@ -0,0 +1,263 @@
+# Ramin Honary
+
+  - Worked at Cross Compass since 2018
+  - Platform Team development, full stack
+  - Worked primarily on Manufacia
+  - Currently on working on ML-Platform
+
+## Why I use PyQt
+
+  - Developed some Python scripts that used **OpenCV**
+      - "Pattern Matching Kit" (will demo)
+      - "Image Cropping Kit" (in progress)
+  - OpenCV provides a API for GUI tools based on PyQt
+  - Decided to use PyQt directly instead of OpenCV specific API
+
+# Part 1: What is Qt
+
+Qt is a cross-platform desktop computing GUI framework
+
+  - Free software (LGPL)
+    - Option of buying commercial licenses
+  - Develops native apps for
+    - Windows, MacOS
+  - Linux
+    - Core of the KDE Plasma desktop environment
```

Comment:
```suggestion
      - Linux
          - Core of the KDE Plasma desktop environment
```

Windows, MaxOS and Linux should be at the same level, aren't they?

Reply:
Agreed